Public/Sdk/Public/Tools/QTest/Tool.QTestRunner.dsc

```diff
@@ -108,6 +108,18 @@ function validateArguments(args: QTestArguments): void {
     }
 }
 
+/**
+ * Find Flaky Supression file from the .config directory of source code
+ */
+function findFlakyFile(): File {
+    let configDir = d`${Context.getMount("SourceRoot").path}\.config`;
+    let flakyFileName = 'CloudBuild.FlakyTests.json';
+    let flakyFiles = globR(configDir, flakyFileName);

```

Comment:
> let flakyFiles = globR(configDir, flakyFileName); [](start = 3, length = 50)

It's best to avoid the glob. Can you just construct the absolute path directly?

Reply:
The path for the file inside .config folder is up to customer's discretion.
The .config directory is quite sparse, so with a full filename specified the globR should return quite quickly.

Reply:
Do we need to give customers the flexibility to place that file at any arbitrary path under their config directory? That makes us add that glob, which could have some performance implications on graph caching if other files in that directory churn. It also creates the problem of error handling if you find 2 files with the same name. What's the upside for BuildXL customers? (I know there probably isn't consistency for QuickBuild customers)

Reply:
Decided to restrict the file location to one of two places. I have pushed the change.